src/server/config.rs

```diff
@@ -105,6 +105,11 @@ pub struct Config {
     pub heavy_load_wait_duration: ReadableDuration,
     pub enable_request_batch: bool,
 
+    // Test only.
+    #[doc(hidden)]
+    #[serde(skip_serializing)]
+    pub raft_client_backoff_step: ReadableDuration,
```

Comment:
`config(skip)` is also necessary.

Reply:
And is it really test only? I found that it's used in `start`.

Reply:
Yes. It's only for test make a shorter backoff. In release mode, the configuration should remain default.